pkg/securityscan/jobHandler.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/rancher/wrangler/pkg/name"
 )
 
+var SonobuoyWorkerLabel = map[string]string{"sonobuoy-plugin": "rancher-kube-bench"}
```

Comment:
This shouldn't be exported, and where does this originally come from? What sets it?

Reply:
It comes from security-scan sonobuoy tool

Reply:
It is set by the security-scan image